src/core/client/stream/tabs/Comments/Stream/SortMenu.css

```diff
@@ -14,6 +14,15 @@
   line-height: 1.14;
 }
 
+@media screen and (max-width: 500px) {
```

Comment:
would it work if you used the `sm` breakpoint variable (640px)?

Reply:
D'oh! Yes, it likely can. Let me fix that up.